umongo/abstract.py

```diff
@@ -181,7 +184,7 @@ def _extract_marshmallow_field_params(self, mongo_world):
         params.update(self.metadata)
         return params
 
-    def as_marshmallow_field(self, params=None, mongo_world=False):
+    def as_marshmallow_field(self, params=None, mongo_world=False, **kwargs):
         """
         Return a pure-marshmallow version of this field.
 
```

Comment:
You should also rename kwargs into field_kwargs here too.

Reply:
Oh, you meant kwargs below, not in the signature. Yes, you're right.